BoxContentSDK/BoxContentSDK/External/KeychainItemWrapper/BOXKeychainItemWrapper.m

```diff
@@ -355,6 +361,11 @@ - (void)writeToKeychain
         }
         result = SecItemAdd((CFDictionaryRef)query, NULL);
         BOXAssert( result == noErr, @"Couldn't add the Keychain Item with error %d.", (int)result);
+        NSDictionary *userInfo = @{@"completion_status" : (result == noErr) ? @"succeeded" : @"failed",
+                                   @"return_code" : @(result)};
```

Comment:
[@(number) stringValue]

Reply:
Not sure I understand the purpose of converting to a string here.